locales/en-US/formautofill.properties

```diff
@@ -3,26 +3,36 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 preferenceGroupTitle = Form Autofill
-enableProfileAutofill = Enable Profile Autofill
-savedProfiles = Saved Profiles…
+enableAddressAutofill = Autofill addresses
+learnMore = Learn more
+savedAddresses = Saved Addresses…
+enableCreditCardAutofill = Autofill credit cards
+savedCreditCards = Saved Credit Cards…
 saveAddressesMessage = Firefox now saves addresses so you can fill out forms faster.
-viewAutofillOptionsLink = View Form Autofill Options
+autofillOptionsLink = Form Autofill Options
+autofillSecurityOptionsLink = Form Autofill & Security Options
```

Comment:
Are these two links? Where are they displayed?

Reply:
They are used as links on the doorhangers, one is for [address](https://mozilla.invisionapp.com/share/AP8TFZ22G#/screens/190747959) and the other for [credit card](https://mozilla.invisionapp.com/share/7ZA4WEK9W#/screens/215537981). Do you think it's a good idea to prefix where the string will be used on all the IDs? (ex. `doorhanger.`, `pref.`, `dropdown.`) Would there be cases where being too specific is bad?

Reply:
No need, explaining how they are used in a localization note is enough.